specs/electra/beacon-chain.md

```diff
@@ -1265,18 +1287,24 @@ def process_attestation(state: BeaconState, attestation: Attestation) -> None:
     # [Modified in Electra:EIP7549]
     assert data.index == 0
     committee_indices = get_committee_indices(attestation.committee_bits)
-    participants_count = 0
-    for index in committee_indices:
-        assert index < get_committee_count_per_slot(state, data.target.epoch)
-        committee = get_beacon_committee(state, data.slot, index)
-        participants_count += len(committee)
+    committee_offset = 0
+    for committee_index in committee_indices:
+        assert committee_index < get_committee_count_per_slot(state, data.target.epoch)
+        committee = get_beacon_committee(state, data.slot, committee_index)
+        committee_attesters = set(
```

Comment:
nit: not that important but this part is quite similar to the logic in `get_attesting_indices()`. I think we can extract common part to a separate util function.

Reply:
I was looking at it as well but it doesn’t seem there is a good way to decompose